ketcher-autotests/tests/User-Interface/Zoom-Changes/zoom-changes.spec.ts

```diff
@@ -44,4 +59,100 @@ test.describe('Zoom changes', () => {
 
     expect(zoomInputValue).toBe('80%');
   });
+
+  test('Zoom In button verification', async ({ page }) => {
+    /*
+    Test case: EPMLSOPKET-1761
+    */
+    await pressButton(page, STRUCTURE_LIBRARY_BUTTON_NAME);
+    await page.getByRole('tab', { name: 'Functional Groups' }).click();
+    await selectFunctionalGroups(FunctionalGroups.CO2Et, page);
+    await clickInTheMiddleOfTheScreen(page);
+    await resetCurrentTool(page);
+
+    await takeTopToolbarScreenshot(page);
+    await takeEditorScreenshot(page);
+
+    const zoomInput = page.getByTestId(TestIdSelectors.ZoomInput);
+    zoomInput.click();
+    await page.getByText('Zoom in').click();
+
+    await takeTopToolbarScreenshot(page);
+    await takeEditorScreenshot(page);
+  });
+
+  test('Zoom Out button verification', async ({ page }) => {
+    /*
+    Test case: EPMLSOPKET-1762
+    */
+    await pressButton(page, STRUCTURE_LIBRARY_BUTTON_NAME);
+    await page.getByRole('tab', { name: 'Functional Groups' }).click();
+    await selectFunctionalGroups(FunctionalGroups.CO2Et, page);
+    await clickInTheMiddleOfTheScreen(page);
+    await resetCurrentTool(page);
+
+    await takeTopToolbarScreenshot(page);
+    await takeEditorScreenshot(page);
+
+    const zoomInput = page.getByTestId(TestIdSelectors.ZoomInput);
+    zoomInput.click();
+    await page.getByText('Zoom out').click();
+
+    await takeTopToolbarScreenshot(page);
+    await takeEditorScreenshot(page);
+  });
+
+  test('Zoom in structure verification', async ({ page }) => {
+    /*
+    Test case: EPMLSOPKET-1763
+    */
+    await selectUserTemplatesAndPlaceInTheMiddle(TemplateLibrary.Azulene, page);
+    await resetCurrentTool(page);
+
+    await takeTopToolbarScreenshot(page);
+    await takeEditorScreenshot(page);
+
+    const zoomInput = page.getByTestId(TestIdSelectors.ZoomInput);
+    zoomInput.click();
+    await page.getByText('Zoom in').click();
+
+    await takeTopToolbarScreenshot(page);
+    await takeEditorScreenshot(page);
+
+    await resetCurrentTool(page);
+    await selectTopPanelButton(TopPanelButton.Undo, page);
+    await takeTopToolbarScreenshot(page);
+    await takeEditorScreenshot(page);
+
+    await resetCurrentTool(page);
+    await selectTopPanelButton(TopPanelButton.Redo, page);
+    await takeTopToolbarScreenshot(page);
+    await takeEditorScreenshot(page);
+
+    await resetCurrentTool(page);
+    await selectTopPanelButton(TopPanelButton.Clear, page);
+    await resetCurrentTool(page);
+
+    await pressButton(page, STRUCTURE_LIBRARY_BUTTON_NAME);
+    await page.getByRole('tab', { name: 'Functional Groups' }).click();
+    await selectFunctionalGroups(FunctionalGroups.CO2Et, page);
+    await clickInTheMiddleOfTheScreen(page);
+    await resetCurrentTool(page);
+
+    await page.keyboard.press('+');
+
+    await page.goto('');
+
+    await openFileAndAddToCanvas('clean_diffproperties.mol', page);
```

Comment:
Let's use common approach for file names. My proposal is to use `-` as separator, just because we name folder this way and playwright itself also generates screenshots with this approach

Reply:
done